tokens/connect/subwallet.md

```diff
@@ -0,0 +1,149 @@
+
+![The main screen of the SubWallet browser extension.](/images/tokens/connect/subwallet/subwallet-1.png)
+
+On the following screen, you'll be prompted to create a password to secure your new wallet:
+
+1. Enter a password that has at least 8 characters
+2. Confirm the password by entering it again
+3. Click **Continue**
+
+![The create a password screen on the SubWallet browser extension.](/images/tokens/connect/subwallet/subwallet-2.png)
+
+You'll then be prompted to back up your seed phrase. This is an important step, especially because you have the option to later derive additional accounts from this seed phrase.
+
+1. View your seed phrase and save it in a safe place
+
+    !!! remember
+        You should never share your seed phrase (mnemonic) or private key with anyone. This gives them direct access to your funds. This guide is for educational purposes only.
+
+2. Once you've safely stored your seed phrase, click **I have kept it somewhere safe**
+
+![Back up your seed phrase on the SubWallet browser extension.](/images/tokens/connect/subwallet/subwallet-3.png)
+
+Once you've created a password and saved your seed phrase, you'll be connected to your account. You can add additional accounts by taking the following steps:
+
+1. Click on the account dropdown
+2. Select one of the options from the bottom of the screen. You can click **Create a new account** to repeat the process you just went through, the import button to import an existing account, or the attach button to attach to an existing account with the private key or seed phrase
+
+![View account details and create a new account, import one, or attach one.](/images/tokens/connect/subwallet/subwallet-4.png)
+
+### Import an Account {: #import-an-account }
+
+To import an existing account into SubWallet, you can select **Import an account**.
+
+![The main screen of the SubWallet browser extension.](/images/tokens/connect/subwallet/subwallet-5.png)
+
+On the following screen, select the method by which you would like to import the existing account. You can choose from **Import from seed phrase**, **Import from Polkadot.{js}**, **Import by MetaMask private key**, and **Import by QR code**.
+
+If you select **Import from seed phrase**, there are some incompatibility issues that can arise when importing an account from seed phrase. For example, Trust Wallet and SafePal are among the wallets not compatible with SubWallet. If you run into incompatibility issues, SubWallet recommends creating a new wallet.
+
+If you select **Import from Polkadot.{js}**, you'll need to make sure that the account was created in Polkadot.js via private key. If it was created with a seed phrase and you attempt to import it to SubWallet, a different public address will be used. This is because Polkadot.js uses BIP39, whereas Ethereum uses BIP32 or BIP44.
```

Comment:
Might be helpful to include links to BIP39 and the others so users can read more about the standards and how they work.